Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImpl.java

```diff
@@ -272,9 +272,9 @@ protected void loadDocumentResponses(ihe.iti.xds_b._2007.RetrieveDocumentSetResp
         }
     }
 
-    protected boolean setDocumentResponse(Document doc, DocumentResponse oDocResponse) {
+    protected boolean setDocumentResponse(DocumentMetadata doc, DocumentResponse oDocResponse) {
         boolean bHasData = false;
-        if (doc.getRawData() != null && doc.getRawData().length > 0) {
+        if (doc.getRawData().length > 0) {
```

Comment:
Make sure we check for NULL

Reply:
I modified this to return a new empty byte array instead of NULL. Gets rid of a branch, and the possibility of getting a NullPointerException.